plugins/modules/ovirt_vm.py

```diff
@@ -911,9 +911,15 @@
         version_added: 1.7.0
     multi_queues_enabled:
         description:
-            - "If `true`, each virtual interface will get the optimal number of queues, depending on the available virtual Cpus."
+            - "If `true` and virtio_scsi_multi_queues parameter is not specified, each virtual interface will get the optimal number of queues,
+              depending on the available virtual Cpus."
         type: bool
         version_added: 1.7.0
+    virtio_scsi_multi_queues:
+        description:
+            - "Number of Virtio SCSI queues used by virtual machine."
```

Comment:
if you're adding the enable/disable option for virtio scsi multi-queue, this should explain the relation between them (e.g., what if the boolean flag is set to enabled while this is set to 0, whether this argument can be set to -1 - such things)

Reply:
Done, what you think now?